go/grpcweb/grpc_web_response.go

```diff
@@ -95,7 +90,7 @@ func (w *grpcWebResponse) finishRequest(req *http.Request) {
 		w.copyTrailersToPayload()
 	} else {
 		w.WriteHeader(http.StatusOK)
-		w.Flush()
```

Comment:
@johanbrandhorst whilst we wait for CI to go green on this PR...

...isn't this the root cause of the issue? Shouldn't this be replaced with a safe up-cast on `w.wrapped` before calling `Flush()`?

Reply:
I haven't looked at it in detail, lets discuss after merging the fix.